tests/Doctrine/Tests/Common/Annotations/AnnotationRegistryTest.php

```diff
@@ -168,4 +168,25 @@ public function testResetClearsRegisteredAutoloaderFailures() : void
 
         self::assertSame(2, $failures);
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testRegisterLoaderIfNotExistsOnlyRegisteresSameLoaderOnce() : void
+    {
+        $className = 'autoloadedClass' . random_int(10, 100000);
```

Comment:
Does it really have to be random? It makes it impossible to reproduce the test case consistently. Please use a fixed class name.

Reply:
I don't mind changing this, but I copied it from this file in `testStopCallingLoadersAfterClassIsFound`.

Reply:
Unless there's a reason the test needs to have a random class name, please change it to a non-random one.